content/chat/replies.textile

```diff
@@ -0,0 +1,122 @@
+---
+title: Replies and Threads
+meta_description: "Implement the ability to reply to or create a thread based on any previously sent messages in your chat application."
+product: chat
+languages:
+  - javascript
+---
+
+Reply to a message directly with a reference to the original message, or create threads on a message to separate the new branch of messages from the main path of messages. Replies enable members to quickly and directly respond to an existing message. Threads enable members to create a thread for a specifically focused conversation between two or more members within the channel.
+
+h2(#enable). Enable replies and threads
+
+To use replies and threads, you'll need to enable this feature as a new "channel rule":/channels#rules within your application in the dashboard.
```

Comment:
I think the name of the feature (Message Interactions) that needs to be enabled should be in this first paragraph.
```suggestion
To use replies and threads, you'll need to enable the "Message Interactions":/channels/messages#interactions feature as a new "channel rule":/channels#rules within your application in the dashboard.
```